java/mykola-melnyk/src/com/zoolatech/lecture1/tasks/_5/Main.java

```diff
@@ -0,0 +1,26 @@
+package com.zoolatech.lecture1.tasks._5;
+
+/**
+ * Write a program that accepts two numbers
+  *      and outputs the larger number.
+   *     If numbers are equal - output “Numbers are equal”.*/
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter first number:");
+        double value1 = scanner.nextDouble();
+        System.out.println("Enter second number:");
+        double value2 = scanner.nextDouble();
+        double largerValue = (value1 > value2)?(value1):(value2);
```

Comment:
I think it would be better to move this assignment into the else block: right now we execute for all cases, if we move it inside - it will be executed only if numbers are not equal.

Reply:
Resolved: Moved line down into else block.